maroto.go

```diff
@@ -191,6 +191,10 @@ func (m *Maroto) addRows(rows ...core.Row) {
 }
 
 func (m *Maroto) addRow(r core.Row) {
+	if len(r.GetColumns()) == 0 {
+		r.Add(col.New(0))
```

Comment:
Bro, I think that this should be 12 and not 0

Reply:
My idea of ​​using '0' was because this would be an empty column, so I thought 0 would make more sense, but in this case 12 would be why?